package.json

```diff
@@ -17,6 +17,7 @@
     "publish-pkg": "./shell/scripts/publish-pkg.sh",
     "serve-pkgs": "./shell/scripts/serve-pkgs",
     "publish-shell": "./shell/scripts/publish-shell.sh",
```

Comment:
Do we intend to remove `publish-shell` in the future?

Reply:
Good point. We can remove the script mention, but we will leave the shell script in